react/features/overlay/functions.ts

```diff
@@ -1,15 +1,27 @@
-import { IReduxState } from '../app/types';
+/* eslint-disable lines-around-comment */
 
-import { getOverlays } from './overlays';
+import { IReduxState } from '../app/types';
 
+// @ts-ignore
+import PageReloadOverlay from './components/web/PageReloadOverlay';
+// @ts-ignore
+import SuspendedOverlay from './components/web/SuspendedOverlay';
+// @ts-ignore
+import UserMediaPermissionsOverlay from './components/web/UserMediaPermissionsOverlay';
 /**
  * Returns the overlay to be currently rendered.
  *
  * @param {IReduxState} state - The Redux state.
  * @returns {?React$ComponentType<*>}
  */
 export function getOverlayToRender(state: IReduxState) {
```

Comment:
Inline this wherever it need to be, functions.ts cannot import web stuff or we are going to break things...